MonoGame.Framework/SDL/SDLGameWindow.cs

```diff
@@ -64,6 +69,11 @@ public override IntPtr Handle
             get { return _handle; }
         }
 
+        private bool HasNativeWindow
```

Comment:
Pointless helper property, the code behind it is simple enough that you should that code.